task/servicetest/servicetest.go

```diff
@@ -174,17 +174,17 @@ func testTaskCRUD(t *testing.T, sys *System) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	f = findTask(fs, task.ID)
+	f = findTask(fs, tsk.ID)
 	found["FindTasks with Organization filter"] = f
 
 	fs, _, err = sys.ts.FindTasks(sys.Ctx, platform.TaskFilter{User: &cr.UserID})
 	if err != nil {
 		t.Fatal(err)
+	} else {
+		f = findTask(fs, tsk.ID)
```

Comment:
This could/should be moved out of the `else` too. `else` looks unusual after a `t.Fatal`.